src/python/WMCore/WMSpec/WMWorkloadTools.py

```diff
@@ -378,6 +378,18 @@ def setAssignArgumentsWithDefault(arguments, argumentDefinition):
 
     return
 
+def validateOutputModules(workload):
+    """
+    Makes sure all output modules comply with DBS restrictions
+    """
+    for task in workload.getAllTasks():
```

Comment:
we should probably define `cpuOnly=True`, to avoid parsing Cleanup and LogCollect jobs.